ChangeLog.md

```diff
@@ -1,5 +1,8 @@
 # eRecruiter API ChangeLog
 
+## 1.58.0
+- `PasswordResponses` Models updated to match eRecruiter requirements 

```

Comment:
please use
- `PasswordResponses` models updated to match new password policy features